### DISCUSSION TITLE
How to make CS Fixer to output issues line by line in a github PR?

### DISCUSSION THREAD

**Discussion by jardayn** [PHP-CS-Fixer/PHP-CS-Fixer#8170] [Q&A]
Essentially is there a way to highlight each error/issue in each file when using PHP CSF in a github action?    

Currently it just outputs into the action's output. 

CodeSniffer apparently has something like that.  

![image](https://github.com/user-attachments/assets/f98f0133-321c-45d5-a92e-7dff9646c062)

P.S. Why is the gitlab output format returning lines like this: 

`[{"check_name":"PHP-CS-Fixer.ordered_class_elements","description":"PHP-CS-Fixer.ordered_class_elements by PHP CS Fixer 3.62.0 (1ce9e24) 7th Gear by Fabien Potencier, Dariusz Ruminski and contributors.","categories":["Style"],"fingerprint":"16cb377110386d6637dd6188931069fd","severity":"minor","location":{"path":"application\/models\/RouteLoader\/RouteLoader.php","lines":{"begin":0,"end":0}}}`

**Comment by Wirone** [comment]
Please see #7171.
